plugins/woocommerce/changelog/50969-dev-deprecate-query

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: tweak
+
+Log deprecation warnings for `Query` class usage.
```

Comment:
I think it could be more specific to say that we're deprecating analytic reports Query class.